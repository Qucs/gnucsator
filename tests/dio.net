# Qucs 0.0.19  /tmp/dio.sch

VProbe:Pr1 _net0 gnd
Diode:D1 _net1 _net0 Is="1e-15 A" N="1" Cj0="10 fF" M="0.5" Vj="0.7 V" Fc="0.5" Cp="0.0 fF" Isr="0.0" Nr="2.0" Rs="0.0 Ohm" Tt="0.0 ps" Ikf="0" Kf="0.0" Af="1.0" Ffe="1.0" Bv="0" Ibv="1 mA" Temp="26.85" Xti="3.0" Eg="1.11" Tbv="0.0" Trs="0.0" Ttt1="0.0" Ttt2="0.0" Tm1="0.0" Tm2="0.0" Tnom="26.85" Area="1.0"
Diode:D2 _net0 gnd Is="1e-15 A" N="1" Cj0="10 fF" M="0.5" Vj="0.7 V" Fc="0.5" Cp="0.0 fF" Isr="0.0" Nr="2.0" Rs="0.0 Ohm" Tt="0.0 ps" Ikf="0" Kf="0.0" Af="1.0" Ffe="1.0" Bv="0" Ibv="1 mA" Temp="26.85" Xti="3.0" Eg="1.11" Tbv="0.0" Trs="0.0" Ttt1="0.0" Ttt2="0.0" Tm1="0.0" Tm2="0.0" Tnom="26.85" Area="1.0"
.DC:DC1 Temp="26.85" reltol="0.001" abstol="1 pA" vntol="1 uV" saveOPs="no" MaxIter="150" saveAll="no" convHelper="none" Solver="CroutLU"
Idc:I1 gnd _net1 I="1 mA"
