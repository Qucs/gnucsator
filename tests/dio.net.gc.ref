<Qucs Dataset 0.0.19>
<indep _net0.V 1>
-0.7146539
</indep>
<indep _net1.V 1>
-1.429308
</indep>
<indep Pr1.V 1>
-0.7146539
</indep>
<indep time 0>
</indep>
