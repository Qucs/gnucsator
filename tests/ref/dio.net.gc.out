default plugins: ignored
;: already installed, replacing
stashing as ;:0
*: already installed, replacing
stashing as *:0
': already installed, replacing
stashing as ':0
": already installed, replacing
stashing as ":0
L: already installed, replacing
stashing as L:0
Gnucap   System status
iterations: op=14, dc=0, tran=0, fourier=0, total=28
transient timesteps: accepted=0, rejected=0, total=0
nodes: user=2, subckt=0, model=0, total=2
dctran density=100.0%, ac density=100.0%

<Qucs Dataset>
<indep _net0.V 1>
-0.7146539
</indep>
<indep _net1.V 1>
-1.429308
</indep>
<indep Pr1.V 1>
-0.7146539
</indep>
